EvidenceApi.Tests/V1/Domain/DocumentSubmissionTests.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using EvidenceApi.V1.Domain;
+using FluentAssertions;
+using NUnit.Framework;
+using AutoFixture;
+using EvidenceApi.V1.Domain.Enums;
+
+namespace EvidenceApi.Tests.V1.Domain
+{
+    [TestFixture]
+    public class DocumentSubmissionTests
+    {
+        private Fixture _fixture = new Fixture();
+
+        [Test]
+        public void DocumentSubmissionHaveCorrectAttributes()
+        {
+            Guid id = new Guid("e2b06785-eb31-4638-8061-e9adad5c430f");
```

Comment:
Generate a new GUID instead of using one that might have some significance
```suggestion
            Guid id = Guid.NewGuid();
```